internal/util/addmergecomment/addmergecomment.go

```diff
@@ -90,8 +89,9 @@ func addUpstreamAnnotation(object *kyaml.RNode, mergeComment string) error {
 	} else if object.GetNamespace() == resid.TotallyNotANamespace {
 		namespace = unknownNamespace
 	}
-	annotations[upstreamIdentifier] = fmt.Sprintf(upstreamIdentifierFmt, group, object.GetKind(), namespace, name)
-	return object.SetAnnotations(annotations)
+	upstreamIdentifierValue := fmt.Sprintf(upstreamIdentifierFmt, group, object.GetKind(), namespace, name)
+	_, err := object.Pipe(kyaml.SetAnnotation(upstreamIdentifier, upstreamIdentifierValue))
```

Comment:
You can just use the `PipeE` function if you just want the error return value.

Reply:
Thanks! Done